api/EXT_blend_minmax.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_blend_minmax",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "38"
```

Comment:
Shipped by https://bugs.chromium.org/p/chromium/issues/detail?id=389728, and the date supports that this was Chrome 38.